### PR TITLE
Adding wearability to fried vox trash

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do
adds the ability to wear empty fried vox, specifically the trash item that you get after eating KFV

## Why It's Good For The Game
Its mindless fun, the game is better when its more fun

## Images of changes
friedvoxel
friedboxel
Corgert

## Changelog
:cl:
add: Adds functionality to fried vox trash so it can be worn on head
tweak: Tweaks the in hand sprites for fried vox
imageadd: Adds sprites for wearing fried vox trash on head, empty fried vox in hand, species specific sprites for drask, skrell, and kidan
/:cl:

### CODE REVIEW

**Comment by Kyep** [comment]
Move over, omnicone.
Here comes... lord omnibucket!

**Comment by TheSardele** [comment]
Keep in mind that certain hats and helmets can look weird on certain species, greys, drask and vox usually. Might want to make sure they look proper on them too.

**Comment by kugamo** [comment]
I am reviewing how they all look on the different species, and I think I will move the sprites back further on the head for the sides

**Comment by taukausanake** [comment]
Don't use a single sprite for all species. We have different sprite sheets in `icons/mob/species/` and a list variable that defines when a specie should use that sprite sheet. You normally have to make special sprites for Vox and Greys, maybe Skrell and Drask, but most human sprites work on other speices. Just check them and if they don't look good on that specie then make a new sprite for them in their `icons/mob/species` file

**Comment by kugamo** [comment]
dreamseeker_iH0BAyUysR
I found that only skrell and drask really needed sprite changes.... Kidan too, but theres no Kidan folder for alt sprites.

**Comment by kugamo** [comment]
dreamseeker_TBQVVE9rwI
ok now I add a folder for kidan specific sprites

**Comment by kugamo** [comment]
friedboxel
Heres what the sprites look like 1 pixel futher back for the side views. I like this better

**Comment by BakKeener** [comment]
I love this and everything it stands for.

**Comment by kugamo** [comment]
dreamseeker_QVXvilAkg7
I altered the drask specific sprite so it wouldnt overlap their top eye. As of now I am finished with this PR, and am happy with how it is.

**Comment by PacifistDalek** [comment]
This is wonderful. My only suggestion is an Ian sprite, perhaps Colonel Ian? Buckethead Ian?

**Comment by kugamo** [comment]
I will leave letting ian wear it to someone else, or me at a later time.

**Comment by kugamo** [comment]
Corgert
Okay I added it to the list of things corgis can wear.

File: code/datums/dog_fashion.dm [hunk 1]
```diff
@@ -200,3 +200,7 @@
 	D.mutations.Add(BREATHLESS)
 	D.atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	D.minbodytemp = 0
+
+/datum/dog_fashion/head/fried_vox_empty
+	name = "REAL_NAME, the number one fan of KFV"
```
Comment (AffectedArc07) [inline code/datums/dog_fashion.dm:205]: Having Ian called

`Ian, the number one fan of KFV` is not only a super long name, but also seems way out of place. Thats more of a description, not a mob name.

Comment (kugamo) [reply code/datums/dog_fashion.dm:205]: what about "#1 KFV fan Ian"

Comment (AffectedArc07) [reply code/datums/dog_fashion.dm]: Honestly I am not sure if he should even be renamed, especially not with the `#1` part.

File: code/datums/dog_fashion.dm [hunk 2]
```diff
@@ -202,5 +202,5 @@
 	D.minbodytemp = 0
 
 /datum/dog_fashion/head/fried_vox_empty
-	name = "REAL_NAME, the number one fan of KFV"
+	name = "Colonel REAL_NAME"
```
Comment (AffectedArc07) [inline code/datums/dog_fashion.dm:205]: Much better

File: code/datums/dog_fashion.dm (hunk 1 above)
Comment (kugamo) [reply code/datums/dog_fashion.dm]: Colonel Ian, _how didnt I think of that._ That should work better.

**Comment by PacifistDalek** [comment]
Yessss my first positive contribution suggestion wise. Looks incredible!!

**Comment by procdrone** [comment]
This is a bad joke. a _terrible_ joke. Revert KFV completly.

**Comment by AffectedArc07** [comment]
@procdrone keep it relevant to the PR. This PR is about the wearability, not the item itself.

**Comment by variableundefined** [comment]
@kugamo Approved. Will merge when deconflicted - pick one of your PR first

**Comment by Kiyahitayika** [comment]
Can we get this sprited for _Vox_ as well?

**Pull request merged**